services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs.py

```diff
@@ -134,358 +119,26 @@ async def get_dynamic_proxy_spec(
-) -> Deque[SimcoreServiceSettingLabelEntry]:
-    """NOTE: this method will modify settings in place"""
-
-    def _format_env_var(env_var: str, destination_container: str) -> str:
-        var_name, var_payload = env_var.split("=")
-        json_encoded = json.dumps(
-            dict(destination_container=destination_container, env_var=var_payload)
-        )
-        return f"{var_name}={json_encoded}"
-
-    for entry in settings:
-        entry: SimcoreServiceSettingLabelEntry = entry
-        if entry.name == "env" and entry.setting_type == "string":
-            # process entry
-            list_of_env_vars = entry.value if entry.value else []
-
-            # pylint: disable=protected-access
-            destination_container = entry._destination_container
-
-            # transforms settings defined environment variables
-            # from `ENV_VAR=PAYLOAD`
-            # to   `ENV_VAR={"destination_container": "destination_container", "env_var": "PAYLOAD"}`
-            entry.value = [
-                _format_env_var(x, destination_container) for x in list_of_env_vars
-            ]
-
-    return settings
-
-
-async def merge_settings_before_use(
-    director_v0_client: DirectorV0Client, service_key: str, service_tag: str
-) -> SimcoreServiceSettingsLabel:
-
-    simcore_service_labels: SimcoreServiceLabels = (
-        await director_v0_client.get_service_labels(
-            service=ServiceKeyVersion(key=service_key, version=service_tag)
-        )
-    )
-    log.info(
-        "image=%s, tag=%s, labels=%s", service_key, service_tag, simcore_service_labels
-    )
-
-    # paths_mapping express how to map dynamic-sidecar paths to the compose-spec volumes
-    # where the service expects to find its certain folders
-
-    labels_for_involved_services: Dict[
-        str, SimcoreServiceLabels
-    ] = await _extract_osparc_involved_service_labels(
-        director_v0_client=director_v0_client,
-        service_key=service_key,
-        service_tag=service_tag,
-        service_labels=simcore_service_labels,
-    )
-    logging.info("labels_for_involved_services=%s", labels_for_involved_services)
-
-    # merge the settings from the all the involved services
-    settings: Deque[SimcoreServiceSettingLabelEntry] = deque()  # TODO: fix typing here
-    for compose_spec_key, service_labels in labels_for_involved_services.items():
-        service_settings: SimcoreServiceSettingsLabel = cast(
-            SimcoreServiceSettingsLabel, service_labels.settings
-        )
-
-        settings.extend(
-            # inject compose spec key, used to target container specific services
-            _add_compose_destination_container_to_settings_entries(
-                settings=service_settings, destination_container=compose_spec_key
-            )
-        )
-
-    settings = _merge_resources_in_settings(settings)
-    settings = _patch_target_service_into_env_vars(settings)
-
-    return SimcoreServiceSettingsLabel.parse_obj(settings)
```

Comment:
love to see all that deleted stuff :rofl:

Reply:
also added missing `_meta.py`